modules/geo-layers/src/mvt-layer/mvt-layer.ts

```diff
@@ -66,11 +66,13 @@ export type MVTLayerProps<DataT = Feature> = TileLayerProps<DataT> & {
   loaders: Loader[];
 };
 
+type MVTParsedTile = Feature[] | BinaryFeatures;
 type ContentWGS84Cache = {_contentWGS84?: Feature[]};
+
 export default class MVTLayer<
   DataT extends Feature = Feature,
   PropsT extends MVTLayerProps<DataT> = MVTLayerProps<DataT>
-> extends TileLayer<DataT, PropsT> {
+> extends TileLayer<DataT, MVTParsedTile, PropsT> {
```

Comment:
Why do we need two data types? Where is `DataT` used?

Reply:
Thanks for link.

I derived typings from behavior of TileLayer and understood, that it is expected that _all_ TileLayer props are propagated to sub layers, including accessors. Now i see this is only unexpected and deprecated side-effect of current implementation.

This example alone probably invalidates renders second dimension useless at all. I'll take a look.

Reply:
I think this can be now resolved (also according with [this discussion](https://github.com/visgl/deck.gl/pull/6842#discussion_r882893537)). The DataT of TileLayer & Tile2dHeader are the same ... and are passed further to renderSubLayers.